SwissTransferFeatures/MainView/STUpdateRequiredView.swift

```diff
@@ -0,0 +1,50 @@
+/*
+ Infomaniak SwissTransfer - iOS App
+ Copyright (C) 2025 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import DesignSystem
+import InfomaniakCoreCommonUI
+import InfomaniakCoreSwiftUI
+import STResources
+import SwiftUI
+import SwissTransferCore
+import VersionChecker
+
+public enum UpdateLink: Sendable {
+    public static let appStore = URL(string: "https://infomaniak.com")!
+    public static let testFlight = URL(string: "https://testflight.apple.com/join/bnHmqCvT")!
+}
+
+public struct STUpdateRequiredView: View {
+    @Environment(\.openURL) private var openURL
+
+    public init() {}
+
+    public var body: some View {
+        UpdateRequiredView(
+            image: STResourcesAsset.Images.documentStarsRocket.swiftUIImage,
+            sharedStyle: TemplateSharedStyle.swissTransfer
+        ) {
+            let url: URL = Bundle.main.isRunningInTestFlight ? UpdateLink.testFlight : UpdateLink.appStore
```

Comment:
```suggestion
            let url = Bundle.main.isRunningInTestFlight ? UpdateLink.testFlight : UpdateLink.appStore
```
We might be able to infer the type here, can't we?